Makefile

```diff
@@ -1,7 +1,15 @@
-testpop:	clean
-	godep go build scripts/testpop/testpop.go
+.PHONY: clean testpop mocks out
+
+GOPATH = $(shell echo $$GOPATH)
+
+out:	testpop
 
 clean:
 	rm -f testpop
 
-.PHONY: clean testpop
+mocks:
+	mockery -dir $(GOPATH)/src/github.com/uber-common/bark -name Logger -output testutils/mocks/ -recursive
```

Comment:
There is no mechanism to change this in mockery. Though you can output the mocks to stdout and pipe them wherever you like.

I think for simplicity we should just stick with mockery's defaults.